velox/functions/prestosql/registration/BinaryFunctionsRegistration.cpp

```diff
@@ -55,6 +55,8 @@ void registerSimpleFunctions(const std::string& prefix) {
       {prefix + "to_base64url"});
   registerFunction<FromBase64UrlFunction, Varbinary, Varchar>(
       {prefix + "from_base64url"});
+  registerFunction<FromBase32Function, Varbinary, Varchar>(
```

Comment:
You need also a function signature for `Varbinary, Varbinary` here. See also `FromBase64Function` to make sense of the template for the call function where the input can be either Varchar or Varbinary.

As a result you also need then tests for the Varbinary input version. See `TEST_F(BinaryFunctionsTest, fromBase64)`.